packages/block-library/src/block/edit.js

```diff
@@ -8,27 +8,30 @@ import classnames from 'classnames';
  */
 import {
 	useEntityBlockEditor,
-	useEntityProp,
 	useEntityRecord,
+	store as coreStore,
 } from '@wordpress/core-data';
 import {
 	Placeholder,
 	Spinner,
-	TextControl,
-	PanelBody,
+	ToolbarButton,
+	ToolbarGroup,
 } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import {
 	useInnerBlocksProps,
 	__experimentalRecursionProvider as RecursionProvider,
 	__experimentalUseHasRecursion as useHasRecursion,
 	InnerBlocks,
-	InspectorControls,
 	useBlockProps,
 	Warning,
 	privateApis as blockEditorPrivateApis,
+	BlockControls,
+	store as editorStore,
```

Comment:
Could we name this `blockEditorStore` instead to not be confused with the store in `@wordpress/editor`? 😅